plaso/output/shared_elastic.py

```diff
@@ -248,6 +258,15 @@ def Close(self):
 
     self._client = None
 
+  def GetClientMajorVersion(self):
```

Comment:
Since this is only used internally please prefix with an underscore

Reply:
done